src/constants/transect_protocols.js

```diff
@@ -0,0 +1,36 @@
+export const protocolsArray = [
```

Comment:
Prefer camelCase or kebab-case to underscore_case when naming JS files. 

CapitalCamelCase when naming components.